coops-ui/src/components/EntityInfo.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  <div class="entity-info" :class="{ 'staff': isRoleStaff }">
+  <div id="entity-info" :class="{ 'staff': isRoleStaff }">
     <v-container>
 
       <!-- Business Name, Business Status -->
       <div class="title-container">
-        <div class="entity-name mb-1">{{ entityName || 'Not Available' }}</div>
-        <v-chip class="entity-status" small label text-color="white" v-if="entityStatus"
+        <div class="mb-1" id="entity-legal-name">
+          <span>{{ entityName || 'Not Available' }}</span>
+        </div>
+        <v-chip id="entity-status" small label text-color="white"
+          v-if="entityStatus"
           :class="{
             'blue' : entityStatus === 'GOODSTANDING',
             'red' : entityStatus === 'PENDINGDISSOLUTION' | 'NOTINCOMPLIANCE',
```

Comment:
I know you didn't touch this code in this PR but possibly a good spot for us to refactor into some Constants or an entityStatus Enum in the future :)

Reply:
Agreed and implemented in a separate commit below.

Reply:
Great! Nice job 👍